controller/handler/trial.js

```diff
@@ -11,6 +11,7 @@ const processComplianceCount = require('../helper/process-compliance-count');
 const processRules = require('../helper/process-rules');
 const processPatientStatus = require('../helper/process-patient-status');
 const httpNotFound = 404;
+const sqlDateFormat = 'ddd MMM DD YYYY HH:mm:ss ZZ';
```

Comment:
Please remove the sqlDateFormat variable. I do not think we require it.
Instead use the below conversions -

patient.dateStarted = moment(patient.dateStarted).format(''YYYY-MM-DD')
patient.dateCompleted = moment(patient.dateCompleted).format('YYYY-MM-DD')

This is the same way for IRB start & end date population in dashboard controller. 
Please refer if required.